test/smart_city/dataset/technical_test.exs

```diff
@@ -56,6 +56,30 @@ defmodule SmartCity.Dataset.TechnicalTest do
       )
     end
 
+    data_test "" do
+      actual =
+        Technical.new(%{
+          dataName: "dataset",
+          orgName: "org",
+          systemName: "org__dataset",
+          sourceUrl: "https://example.com",
+          sourceFormat: extension
+        })
+
+      assert Map.get(actual, :sourceFormat) == mime_type
+
+      where(
+        extension: ["gtfs", "csv", "zip", "kml", "json"],
```

Comment:
I haven't seen us use Checkov this way where we have a list of extensions then a list of mime_types and they are put together based on order.  Would you mind switching it so that "gtfs" and "application/gtfs+protobuf" are next to each other?

Reply:
Do we need to test geo+json mime type with .geojson extensions?

Reply:
how do you mean "next to each other"? they were horizontally next to each other until the formatter decided the line was too long.

Reply:
if you mean do we want to do an exhaustive test of all our supported types @rajones-hntb i didn't think it was necessary but i can if others would like to cover all those bases. this was meant to be representative.

Reply:
I feel like this approach is clearer:
```
      where(
        [:extension, :mime_type],
        ["gtfs", "application/gtfs+protobuf"],
        ["csv", "text/csv"],
        ["zip", "application/zip"],
        ["kml", "application/vnd.google-earth.kml+xml"],
        ["json", "application/json"]
      )
```

Reply:
Not according to Johnson, which is why it's in there

Reply:
i think i would normally tend to agree that the different scenarios are more obvious (i think the nested lists could throw some people) but in this case I'm inclined to go with @JarredOlson since the formatter is decoupling the pairing between the "rows" of inputs.